frontend/pages/post/[id].js

```diff
@@ -398,6 +409,50 @@ class LongPost extends React.Component {
       })
   }
 
+  deleteComment = (postID, commentID) => {
+    this.context.actions
+      .deleteComment(postID, commentID)
+      .then(response => {
+        M.toast({
+          html: 'Successfully Deleted Comment',
+          classes: 'rounded green'
+        })
+        this.setState(prevState => ({
+          comments: prevState.comments.filter(
+            comment => comment._id != commentID
+          )
+        }))
+      })
+      .catch(err => {
+        M.toast({ html: 'Oops, Something Went Wrong', classes: 'rounded red' })
+        console.log(err)
+        //(err)
+      })
+  }
+  editComment = async (postID, commentID, comment) => {
+    await this.context.actions
+      .editComment(postID, commentID, comment)
+      .then(response => {
+        M.toast({
+          html: 'Successfully Edited Comment',
+          classes: 'rounded green'
+        })
+        let comments = this.state.comments
+        for (let i = 0; i < comments.length; i++) {
+          if (comments[i]._id == commentID) {
+            comments[i].content = comment
+          }
```

Comment:
you could use the ES6 Syntax `forEach` or the even better [`findIndex`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex) I think this will make it look better

```js
 const index = comments.findIndex ( comment => comment._id == commentID);
 comments[index].content = comment
```

Reply:
Done.